files/__addonLocation__/gitignore

```diff
@@ -5,8 +5,13 @@
 /LICENSE.md
```

Comment:
a gitignore file (especially this one) must not assume a parent gitignore exists.